3rdparty/openssl/unsupported.h

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#ifndef _OE_OPENSSL_UNSUPPORTED_H
+#define _OE_OPENSSL_UNSUPPORTED_H
+
+#if !defined(OE_OPENSSL_SUPPRESS_UNSUPPORTED)
+
+#if defined(__cplusplus)
+#define OE_OPENSSL_EXTERN_C_BEGIN \
+    extern "C"                    \
+    {
+#define OE_OPENSSL_EXTERN_C_END }
+#else
+#define OE_OPENSSL_EXTERN_C_BEGIN
+#define OE_OPENSSL_EXTERN_C_END
+#endif
+
+#define OE_OPENSSL_UNSUPPORTED(MSG) __attribute__((deprecated(MSG)))
```

Comment:
bits/defs.h has this:
```
#if defined(__GNUC__)
#define OE_DEPRECATED(FUNC, MSG) FUNC __attribute__((deprecated(MSG)))
#elif defined(_MSC_VER)
#define OE_DEPRECATED(FUNC, MSG) __declspec(deprecated(MSG)) FUNC
#else
#define OE_DEPRECATED(FUNC, MSG) FUNC
#endif 
```

But here only has the _GNUC_ line.  Can you confirm that either:
a) this was not intentional and update this PR to make it match, OR
b) this header will not appear in the OE SDK package?

Reply:
Updated. Given that we don't support compiling enclaves with MSVC, I skipped the check.